resources/core/charts/jaeger/templates/virtualservice.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.global.isLocalEnv }}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: core-{{ .Chart.Name }}
```

Comment:
shouldn't be just: {{ template "fullname" . }} ?

Reply:
If changed, please ensure the service name is changed in istio-charts.

Reply:
it has not been changed